From 13e510fc7c107668e9acfd226ff519a480e4a158 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 23 Sep 2018 13:23:42 +0000
Subject: [PATCH 2/4] Allow library flags to be present in LDFLAGS

---
 deps/c-ares/configure | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/deps/c-ares/configure b/deps/c-ares/configure
index b3b27188..a0b15b5d 100755
--- a/deps/c-ares/configure
+++ b/deps/c-ares/configure
@@ -4488,9 +4488,6 @@ printf "%s\n" "$as_me: LIBS error: LIBS may only be used to specify libraries (-
       -I*)
         xc_bad_var_ldflags=yes
         ;;
-      -l* | --library=*)
-        xc_bad_var_ldflags=yes
-        ;;
     esac
   done
   if test $xc_bad_var_ldflags = yes; then
-- 
2.42.0

